bindings/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,388 @@
+be of interest or that has already been addressed.
+
+## Supported Interpreters
+
+PyMongoArrow supports CPython 3.7+ and PyPy3.8+. Language features not
+supported by all interpreters can not be used.
+
+## Style Guide
+
+PyMongo follows [PEP8](http://www.python.org/dev/peps/pep-0008/)
+including 4 space indents and 79 character line limits.
+
+## General Guidelines
+
+-   Avoid backward breaking changes if at all possible.
+-   Write inline documentation for new classes and methods.
+-   Write tests and make sure they pass (make sure you have a mongod
+    running on the default port, then execute `tox -e test` from the cmd
+    line to run the test suite).
+-   Add yourself to doc/contributors.rst `:)`
+
+## Authoring a Pull Request
+
+**Our Pull Request Policy is based on this** [Code Review Developer
+Guide](https://google.github.io/eng-practices/review)
+
+The expectation for any code author is to provide all the context needed
+in the space of a pull request for any engineer to feel equipped to
+review the code. Depending on the type of change, do your best to
+highlight important new functions or objects you've introduced in the
+code; think complex functions or new abstractions. Whilst it may seem
+like more work for you to adjust your pull request, the reality is your
+likelihood for getting review sooner shoots up.
+
+**Self Review Guidelines to follow**
+
+-   If the PR is too large, split it if possible.
+
+    - Use 250 LoC (excluding test data and config changes) as a
+            rule-of-thumb.
+
+     - Moving and changing code should be in separate PRs or commits.
+
+        -   Moving: Taking large code blobs and transplanting
+            them to another file. There\'s generally no (or very
+            little) actual code changed other than a cut and
+            paste. It can even be extended to large deletions.
+        -   Changing: Adding code changes (be that refactors or
+            functionality additions/subtractions).
+        -   These two, when mixed, can muddy understanding and
+            sometimes make it harder for reviewers to keep track
+            of things.
+
+-   Prefer explaining with code comments instead of PR comments.
+
+**Provide background**
+
+-   The PR description and linked tickets should answer the "what" and
+    "why" of the change. The code change explains the "how".
+
+**Follow the Template**
+
+-   Please do not deviate from the template we make; it is there for a
+    lot of reasons. If it is a one line fix, we still need to have
+    context on what and why it is needed.
+
+-   If making a versioning change, please let that be known. See examples below:
+
+    -   `versionadded:: 3.11`
+    -   `versionchanged:: 3.5`
+
+**Pull Request Template Breakdown**
+
+-  **Github PR Title**
+
+    -   The PR Title format should always be
+            `[JIRA-ID] : Jira Title or Blurb Summary`.
+
+-  **JIRA LINK**
+
+-   Convenient link to the associated JIRA ticket.
```

Comment:
--   Convenient link to the associated JIRA ticket.
+    -   Convenient link to the associated JIRA ticket.

(Fixes indent)

Reply:
Fixed